drivers/sensor/ina260/CMakeLists.txt

```diff
@@ -6,3 +6,5 @@
 zephyr_library()
 
 zephyr_library_sources(ina260.c)
+
+zephyr_include_directories(.)
```

Comment:
Don't think this is necessary

Reply:
While ina260.c can access ina260.h without this line, if this isn't included, application code does not get access to the ina260.h. After that discussion about saving the 'raw' driver values for the reduced size, I figured it would be good practice to expose those types.